gitgud/skills/util.py

```diff
@@ -201,13 +201,12 @@ def __init__(self, names, items):
         self._items = items
     
     def __getitem__(self, query):
-        if isinstance(query, int):
-            return self._items[query]
-        elif isinstance(query, str):
+        if isinstance(query, str):
             if query.isnumeric():
-                if int(query) == 0:
+                if 0 < int(query) <= len(self):
+                    return self._items[int(query) - 1]
+                else:
                     raise KeyError
-                return self._items[int(query) - 1]
             return self._items[self._name_dict[query]]
         else:
             raise ValueError('Bad key type.')
```

Comment:
Since we're only accepting strings now, it makes sense to call this a KeyError with no arguments just like we do on above.